Packs/Cisco-umbrella/Integrations/Cisco-umbrella-investigate/Cisco-umbrella-investigate.py

```
@@ -1218,9 +1210,8 @@ def get_domains_for_email_registrar_command():
     # user input validation
     if not isinstance(emails, list):
         return_error('Emails list is not formatted correctly, please try again.')
-    if sort:
-        if sort != 'created' and sort != 'updated':
-            return_error('The parameter sort accept only these values: created/updated/expired.')
+    if sort and sort != 'created' and sort != 'updated':
+        return_error('The parameter sort accept only these values: created/updated/expired.')
```

```suggestion
    if sort and sort not in ['created', 'updated']:
        return_error('The parameter sort accept only these values: created/updated/expired.')
```

---

and let's throw an error and catch from the main()